src/tox_travis/after.py

```diff
@@ -0,0 +1,150 @@
+"""Add a flag to pause and wait for all Travis jobs to complete."""
+from __future__ import print_function
+import os
+import sys
+import json
+import time
+import py
+
+from tox.config import _split_env as split_env
+try:
+    import urllib.request as urllib2
+except ImportError:
+    import urllib2  # Python 2
+
+from .utils import TRAVIS_FACTORS, parse_dict
+
+
+def travis_after_monkeypatch():
+    """Monkeypatch the Tox session to wait for jobs to finish."""
+    import tox.session
+    real_subcommand_test = tox.session.Session.subcommand_test
+
+    def subcommand_test(self):
+        retcode = real_subcommand_test(self)
+        if retcode == 0 and self.config.option.travis_after:
+            travis_after()  # No need to run if the tests failed anyway
+        return retcode
+    tox.session.Session.subcommand_test = subcommand_test
+
+
+def travis_after():
+    """Wait for all jobs to finish, then exit successfully."""
```

Comment:
Possibly, I think that likely makes some sense. This keeps the error codes all in the same function, but constants may be a better way to keep those together anyway.

This was written as a port of `travis_after_all`, which does all of it's configuration based on environment variables. I wonder if it would be better to pull as much configuration as possible into the `[travis:after]` section. Specifically, the API URL and the polling interval, because the rest are native Travis environment variables, or the `GITHUB_TOKEN`, which should be an environment variable so that it can be encrypted in the `.travis.yml`.

The remaining configuration shouldn't